modulus/launch/logging/wandb.py

```diff
@@ -94,7 +94,7 @@ def initialize_wandb(
         wandb_name = f"{name}_{time_string}"
 
     if not os.path.exists(wandb_dir):
-        os.makedirs(wandb_dir)
+        os.makedirs(wandb_dir, exist_ok=True)
```

Comment:
oddly I encountered race conditions here without this flag `exist_ok=True`

Reply:
I've seen this as well. There should be a check to make sure only one process is in charge of initialization. I don't know where the responsibility should be for that though.

Reply:
@daviddpruitt  its a bit tricky as the validation set mightn't be specific years, people might have a random sample of the data s validation and the exact details would be in the dataloader. I think I would leave the current flag but we can change this in the future

Reply:
Thanks for checking with me, I leave that up to you and Jussi to decide